nixos/modules/services/printing/cupsd.nix

```diff
@@ -305,6 +321,16 @@ in
                 [ -L "$i" ] && rm "$i"
```

Comment:
This line is not part of the changeset. Is there any reason you mention this as part of this PR?

according to <https://ss64.com/bash/rm.html>:

> The rm command accepts the -- option which will cause it to stop processing flag options from that point forward.
> This allows the removal of file names that begin with a dash -.
> rm -- -filename
> Alternatively use an absolute or relative path reference.
> rm /home/user/-filename
> rm ./-filename 

As I understand it, the `--` add no value here as `i` will always be filled with an absolute path of the form `/var/lib/cups/*`.